TangemSdk/TangemSdk/TangemSdk.swift

```diff
@@ -623,14 +647,36 @@ extension TangemSdk {
             completion(.failure(error.toTangemSdkError()))
             return
         }
-        
+
         configure()
         cardSession = makeSession(with: config,
-                                  cardId: cardId,
+                                  sessionFilter: sessionFilter,
                                   initialMessage: initialMessage,
                                   accessCode: accessCode)
         cardSession!.start(with: runnable, completion: completion)
     }
+
+    /// Allows running a custom bunch of commands in one NFC Session by creating a custom task. Tangem SDK will start a card session, perform preflight `Read` command,
+    /// invoke the `run ` method of `CardSessionRunnable` and close the session.
+    /// You can find the current card in the `environment` property of the `CardSession`
+    /// - Parameters:
+    ///   - runnable: A custom task, adopting `CardSessionRunnable` protocol
+    ///   - cardId: CID, Unique Tangem card ID number. If not nil, the SDK will check that you tapped the  card with this cardID and will return the `wrongCard` error' otherwise
```

Comment:
🪟
```suggestion
    ///   - cardId: CID, Unique Tangem card ID number. If not nil, the SDK will check that you tapped the card with this cardID and will return the `wrongCard` error otherwise
```